src/test/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumOutputCollectorTest.java

```diff
@@ -45,16 +46,22 @@ public void setUp() {
 
     @Test
     public void collectorWillReturnDefaultOfNegOneObjectiveValueIfLabelNotFound() {
-        Job job = new Job(new FBAParameters(false, false, false));
-        FBAOutput result = (FBAOutput) classToTest.collectOutput(job, mockFBACard);
+        Job job = new Job(new FBAParameters(false, false, false,
+                0, 0, 0, 0, 0,
+                0, 0, 0));
+        FBAOutput result = (FBAOutput) classToTest.collectOutput(job);
+       // FBAOutput result = (FBAOutput) classToTest.collectOutput(job, mockFBACard);
```

Comment:
Why did you comment the second parameter `mockFBACard`?

I can build this successfully with Maven, but when I try to run in Intellij it I get errors from these lines. That seems like a potential problem with our Maven pipeline. @jakedv24

Reply:
Swapped the second and first parameters Daniel, I thought it was outdated originally, but we do need that in fact. These changes should be done now